.github/workflows/ubuntu.yml

```diff
@@ -208,6 +199,15 @@ jobs:
         batch-name: adequacy-patch-CSR
         os: ${{ matrix.test-platform }}
 
+    - name: Run parallel tests
+      if: ${{ env.RUN_EXTENDED_TESTS == 'true' }}
```

Comment:
When is "RUN_EXTENDED_TESTS" true (or false) ?

Reply:
parallel tests takes the same time as short tests, we decided it's shouldn't be run for each job to save time